src/components/pages/IDPManagement/IDPList.tsx

```diff
@@ -94,9 +95,19 @@ export const IDPList = ({ isManagementOSP }: { isManagementOSP?: boolean }) => {
     .sort((a: IdentityProvider, b: IdentityProvider) =>
       (a?.displayName ?? '').localeCompare(b.displayName ?? '')
     )
+  const [idpsManagedData, setIdpsManagedData] = useState(
+    idpsData?.filter((a) => a.identityProviderTypeId === IDPCategory.MANAGED)
```

Comment:
this line is used at least two more times below so let's wrap it in a function to reduce redundancy